corehq/apps/hqwebapp/static/preview_app/scss/preview_app/module.scss

```diff
@@ -45,18 +45,15 @@
   }
 }
 
-.module-go-container {
-  width: 140px;
-}
-
+// todo B5: pagination looks terrible in app preview, not overriding -lg styles properly
```

Comment:
Is there a plan to address this? What's the expected timeline?

Reply:
oops, that's leftover. This PR addresses that todo.